static/js/containers/DashboardPage_test.js

```diff
@@ -666,9 +674,14 @@ describe('DashboardPage', () => {
       let dashboardResponse = {"programs": [ program ]};
       let coursePrices = makeCoursePrices(dashboardResponse);
       let availablePrograms = makeAvailablePrograms(dashboardResponse);
+      let programLearners = makeProgramLearners();
       helper.dashboardStub.returns(Promise.resolve(dashboardResponse));
       helper.programsGetStub.returns(Promise.resolve(availablePrograms));
       helper.coursePricesStub.returns(Promise.resolve(coursePrices));
+      helper.programLearnersStub = helper.fetchJSONWithCSRFStub.withArgs(
+        `/api/v0/programlearners/${program.id}/`
+      );
+      helper.programLearnersStub.returns(Promise.resolve(programLearners));
```

Comment:
it would probably be good to have at least one test making sure that the card doesn't show up in the API doesn't return anything